nexus/tests/integration_tests/schema.rs

```diff
@@ -456,29 +470,50 @@ const REFERENTIAL_CONSTRAINTS: [&'static str; 8] = [
 const VIEWS: [&'static str; 4] =
     ["table_catalog", "table_schema", "table_name", "view_definition"];
 
-const STATISTICS: [&'static str; 8] = [
+const STATISTICS: [&'static str; 11] = [
     "table_catalog",
     "table_schema",
     "table_name",
     "non_unique",
     "index_schema",
     "index_name",
+    "seq_in_index",
     "column_name",
     "direction",
+    "storing",
+    "implicit",
 ];
 
+const PG_INDEXES: [&'static str; 5] =
+    ["schemaname", "tablename", "indexname", "tablespace", "indexdef"];
```

Comment:
Note: These are not actually from "InformationSchema", but rather, from https://www.cockroachlabs.com/docs/stable/pg-catalog .

This comparison was necessary to catch the differing "WHERE" clauses.